Lib/_colorize.py

```diff
@@ -110,3 +125,28 @@ def can_colorize(*, file: IO[str] | IO[bytes] | None = None) -> bool:
         return os.isatty(file.fileno())
     except io.UnsupportedOperation:
         return hasattr(file, "isatty") and file.isatty()
+
+
+def set_theme(t: dict[ColorTag, str] | None = None) -> None:
+    global theme
+
+    if t:
+        theme = t
+        return
+
+    colors = get_colors()
+    theme = {
+        "PROMPT": colors.BOLD_MAGENTA,
+        "KEYWORD": colors.BOLD_BLUE,
+        "BUILTIN": colors.CYAN,
+        "COMMENT": colors.RED,
+        "STRING": colors.GREEN,
+        "NUMBER": colors.YELLOW,
+        "OP": colors.RESET,
+        "DEFINITION": colors.BOLD_WHITE,
```

Comment:
Looks good on macOS with dark theme:

<img width="237" alt="image" src="https://github.com/user-attachments/assets/b924c5f9-4def-41c3-93d1-f86744101ef5" />

But I think we should avoid bold _white_ for light theme:

<img width="235" alt="image" src="https://github.com/user-attachments/assets/f01e355a-12c1-4353-9662-35e5c7f98b56" />

Instead, we can add `BOLD = "\x1b[1m"` to `ANSIColors` with no specific colour.

```suggestion
        "DEFINITION": colors.BOLD,
```

Then we get a bold version of whatever colour the user's terminal has set:

<img width="122" alt="image" src="https://github.com/user-attachments/assets/627127da-1072-454b-951b-02147875db3d" />

<img width="124" alt="image" src="https://github.com/user-attachments/assets/29c9857b-3b47-4887-8144-ad559483005b" />